runtime/help/options.md

```diff
@@ -462,6 +462,17 @@ Here are the available options:
 
     default value: `false`
 
+* `truecolor`: controls the usage of the true color support/mode.
+   * `auto`: will evaluate if the terminal is capable to handle true color.
```

Comment:
"enable usage of true color if micro detects that it is supported by the terminal, otherwise disable it"?